app/javascript/searchable.vue

```diff
@@ -37,6 +37,13 @@
             .card-list-item-meta__item(
               v-if='!["practice", "page", "user"].includes(searchable.model_name)'
             )
+              a.a-img-user-icon(:href='userUrl')
+                img.user-icon.a-user-icon(
+                  :src='searchable.avatar_url',
+                  :title='searchable.icon_title',
+                  :alt='searchable.icon_title',
+                  :class='`is-${searchable.primary_role}`'
```

Comment:
既に定義されているので、DRYなコードにするためにも下記のようにした方がいいと感じました。

```suggestion
                  :class='roleClass'
```

Reply:
@daiki0381 
返信が遅くなってすみません🙏訂正しましたので、確認お願いします。

モデルが違うので、既存の`roleClass`は使えないかと思っていました。ありがとうございます。